src/serialization.h

```diff
@@ -50,23 +50,30 @@
 	28: Added "private" flag to NodeMetadata
 	29: Switched compression to zstd, a bit of reorganization
 */
+
 // This represents an uninitialized or invalid format
-#define SER_FMT_VER_INVALID 255
+constexpr u8 SER_FMT_VER_INVALID = 255;
```

Comment:
Note to reviewers: When using GCC, an `enum Name : u8` would not raise a `-Woverflow` warning in case of an out-of-range value. However, `constexpr u8` does, hence used this notation.